services/queue/config.yml

```diff
@@ -100,6 +100,8 @@ defaults:
     useCloudMirror:               !env:bool:optional USE_CLOUD_MIRROR
     cloudMirrorHost:              'cloud-mirror.taskcluster.net'
 
+    taskCacheMaxSize: 5000
```

Comment:
To make this changeable without releasing a new TC, it needs to be an !env:.. variable.  I think that can use `optional` too, with a suitable default value.